tests/test_duck_duck_go_search.py

```diff
@@ -0,0 +1,21 @@
+"""
+This module contains tasks
+"""
+from playwright.sync_api import Page
+from pytest import mark
+
+from actors.actor import Actor
```

Comment:
کلیتا بهتره هر چیزی یه پکیج به بیرون اکسپوز می‌:نه توی اینیتش باشه و از داخلش چیزی ایمپورت نشه. یعنی 
```suggestion
from actors import Actor
```

Reply:
Done